### DISCUSSION TITLE
! shortcut not working VS Code

### DISCUSSION THREAD

**Discussion by NellyisDevv** [community/community#20407] [Projects and Issues] [answered]
For some reason the ! shortcut for the HTML boilerplate is not working on VS Code, everything else seem's to be working just fine, is this happening to anyone else?

**Comment by FahdSeddik** [comment, marked as answer]
Hey!
You can display IntelliSense by using the shortcut Ctrl+Space after writing "!". Hope this helps!

**Reply by NellyisDevv** [reply]
Thanks so much! I have no Idea why that changed on my end and it was annoying me so much.
